.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+### Are there any dependencies that need to be addresses before this PR is merged?
```

Comment:
what kind of dependencies are we targeting here? software/library ones, people ones? Maybe it's intentionally vague to address either and I'm ok with that, but it is not clear to me if it is meant to be more specific.

Reply:
We've intentionally left these pretty open ended in order to see what kind of use patterns people find natural, with the understanding that it's quick and easy to update the template in the future if we find it's not fitting our needs. We had both software and people dependencies in mind.